stablehlo/transforms/StablehloLegalizeToVhlo.cpp

```diff
@@ -247,18 +267,30 @@ class StablehloToVhloOpConverter : public OpConversionPattern<StablehloOpTy> {
 
     SmallVector<NamedAttribute> vhloAttrs;
     for (NamedAttribute stablehloAttr : stablehloOp->getAttrs()) {
+      Attribute vhloAttr;
       if constexpr (std::is_same<StablehloOpTy,
                                  stablehlo::CustomCallOp>::value) {
         if (stablehloAttr.getName() == "api_version") {
-          auto vhloAttr = convertCustomCallApiVersion(stablehloAttr.getValue());
+          vhloAttr = convertCustomCallApiVersion(stablehloAttr.getValue());
+          if (!vhloAttr) return failure();
+        }
+        if (stablehloAttr.getName() == "called_computations") {
+          vhloAttr = convertCustomCallCalledComputations(
+              stablehloAttr.getValue(), this->getTypeConverter());
+          if (!vhloAttr) return failure();
+        }
+      } else if constexpr (std::is_same<StablehloOpTy, func::CallOp>::value) {
+        if (stablehloAttr.getName() == "callee") {
+          vhloAttr = convertFuncCallee(stablehloAttr.getValue(),
+                                       this->getTypeConverter());
```

Comment:
For the StableHLO --> VHLO direction we don't need to be this precise and can map all FlatStymbolRefAttrs to StringAttr I believe, the other direction will require this though.

Reply:
Probably safe to keep these in sync actually. We will only support flat symbol refs in these ops which is probably best.

Reply:
Alright, will keep the code unchanged then.